### DISCUSSION TITLE
Seeking New Maintainers for This Repository

### DISCUSSION THREAD

**Discussion by HenryQW** [HenryQW/Awesome-TTRSS#486] [Announcements]
Dear fellow RSS enthusiasts,

Unfortunately, due to personal circumstances (busy work, auh.), I am unable to commit enough time to maintain this repository to a satisfactory standard.

### Background of This Repo

The original motivation behind creating this repository was to simplify the deployment process for TTRSS, which did not support Docker deployment back in 2017. I [forked this repo](https://github.com/HenryQW/Awesome-TTRSS/commit/0b4be04fae42a0165a86e501098c81c2d2614c40) from @rubenv <https://github.com/rubenv/docker-ttrss> to offer a more user-friendly solution. I then bundled plugins I deemed useful, I hope you find them useful too ⭐️.

Since then, TTRSS has gradually added Docker support through docker-compose.yml, which in my opinion, might not be the best approach for updating a containerized application, but it works in a much more stable way.

This repo, as it simply wraps TTRSS upstream and containerizes the codebase, always suffers from instability issues, e.g, breaking changes, dependency compatibility. Thus requires effort to maintain it, which is something I am no longer able to provide due to personal circumstances (I've relocated to 3 countries since 2017, wow! )
 
### Some Stats
Over the past 6 years:

Over 8.1M docker pulls
<img width="926" alt="image" src="https://github.com/HenryQW/Awesome-TTRSS/assets/5896343/8e1595e7-3108-4dcd-9442-3f51d119bad7">

Over 130K visits to the doc site
<img width="1570" alt="image" src="https://github.com/HenryQW/Awesome-TTRSS/assets/5896343/0392bb48-c3e7-4ef2-9d1a-8d69e270c209">

As the saying goes, you either grow with your users or grow old with them. In this case, I find myself growing old with all of you ❤️.

To date, I still find RSS to be the best way to acquire information from this increasingly fragmented, closed-off, algorithmically polluted internet. I spend at least an hour reading my RSS feed every day. That's why I hope this repository can continue to serve everyone for as long as it's able.

### Maintainer Resposibility

Make sure the docker image continues to work:
1. Keep up with the upstream updates
2. Improve where necessary, but keep it stable
3. Bug fixes

### If you are interested in taking up the task of maintaining this repository, I would be very grateful. Please leave a message here to express your interest.

Long live RSS!

**Comment by rubenv** [comment]
@HenryQW happy to have indirectly contributed to this all! 😅

**Comment by dimon222** [comment]
Perhaps worth to research if we can have best of two worlds - base off official docker image with any extra tweaks on top that are specific to Awesome TTRSS this way reducing maintenance burden to minimum? Up to "plugins only" perhaps.

**Reply by HenryQW** [reply]
Yes I agree this is a great approach.

**Comment by dimon222** [comment]
I suppose @TonyRL took over and this can be closed now? Kudos Tony for keeping this project going!
But nevertheless I still feel it might be more sustainable longer term to try approach I suggested above - try to base it on top of official image of Tiny Tiny RSS with only "Awesome" modules on top. Its very subjective opinion of course. But it might make things muuuch simpler to maintain. I have my own private fork on top of Full Text RSS (paid one, of course) instead of Mercury. Sadly I can't contribute my config because I see it will change the way too many things right now, but if this was more modular/smaller footprint of repository it might have got easier.

**Reply by TonyRL** [reply]
The [official image setup](https://tt-rss.org/wiki/InstallationNotes#docker-compose.yml) is not a direct replacement for the current Awesome-TTRSS image. Additionally, requiring users to set up two additional containers does not really fit the "Dockerized all-in-one solution".

**Reply by dimon222** [reply]
I was in no way saying it is a replacement. I was suggesting it to use as "base" image (FROM statement in dockerfile) and add "awesomeness" on top. This will ensure closer compatibility, reduce maintenance burden.

**Reply by HenryQW** [reply]
I think it's a good approach. I remember that image was using git pull inside a running container to perform updates, so it might not work.

**Reply by ylavi** [reply]
> The [official image setup](https://tt-rss.org/wiki/InstallationNotes#docker-compose.yml) is not a direct replacement for the current Awesome-TTRSS image. Additionally, requiring users to set up two additional containers does not really fit the "Dockerized all-in-one solution".

Hi

I'm trying to understand whether there is a benefit to using Awesome-TTRSS over the official TTRSS Docker setup.
It seems to me from the discussion here and "About" in the docs ("with enhanced user experience via simplified deployment and a list of curated plugins") that there are definite advantages, but it's a bit vague at that point.

- "Simplified deployment" implies "custom TTRSS image and Docker images for dependencies"?
- "**_list_** of curated plugins" implies preinstalled plugins?
- In what ways does the custom TTRSS image differ from the standard one? Just the plugins or other things?

I'd much appreciate clarification on those fronts so I can decide how I'd like to manage my new instance.

Thanks

(Apologies for asking this question specifically here in a sort-of-unrelated discussion but I know of no way to ask anyone directly and suspect that a new discussion may not be responded to)
